features/google/src/components/importExplanationExpandable/importExplanationExpandable.jsx

```diff
@@ -177,39 +167,44 @@ const ImportExplanationExpandable = ({
                     alt="import-illustration"
                     className="full-screen"
                 />
-                <p>
-                    After you have downloaded your data file onto your phone you
-                    can import it into your polyPod.
-                </p>
-                <div className="separator"></div>
+                <p>{i18n.t("import:import.1")}</p>
+                <InfoBox textContent={i18n.t("import:import.info")} />
+                <div className="poly-separator"></div>
                 <div className="x-divider">
                     {files?.length ? (
                         <div className="file-info">
-                            <h5>Selected file:</h5>
-                            <p>Name: {files[0]?.name}</p>
+                            <h5>{i18n.t("import:import.chosen")}</h5>
                             <p>
-                                Size:
+                                {i18n.t("import:import.name")} {files[0]?.name}
+                            </p>
+                            <p>
+                                {i18n.t("import:import.size")}
                                 {formatSize(files[0]?.size)}
                             </p>
                         </div>
                     ) : selectedFile ? (
                         <div className="file-info">
-                            <h5>Selected file:</h5>
-                            <p>Name: {selectedFile?.name}</p>
+                            <h5>{i18n.t("import:import.chosen")}</h5>
+                            <p>
+                                {i18n.t("import:import.name")}{" "}
+                                {selectedFile?.name}
+                            </p>
                             <p>
-                                Size:
+                                {i18n.t("import:import.size")}
```

Comment:
In this case, if a template had been used, the number would have followed l12n transformation. You can still use it if you want with the `l12n` class.